sycl/test/sub_group/info.cpp

```diff
@@ -72,6 +72,18 @@ int main() {
     /* Sub-group size is not specified in kernel or IL*/
     exit_if_not_equal<size_t>(Res, 0, "compile_num_sub_groups");
 
+    // According to specification, this kernel query requires `cl_khr_subgroups`
+    // or `cl_intel_subgroups`
+    if ((Device.has_extension("cl_khr_subgroups") ||
+         Device.has_extension("cl_intel_subgroups")) &&
+        Device.has_extension("cl_intel_required_subgroup_size")) {
+      Res = Kernel.get_sub_group_info<
+          info::kernel_sub_group::compile_sub_group_size>(Device);
+
+      /* Required sub-group size is not specified in kernel or IL*/
+      exit_if_not_equal<size_t>(Res, 0, "compile_sub_group_size");
```

Comment:
@shileiti, could you please add one more test case where `reqd_sub_group_size` was specified for kernel?

I think that you can create new program using `build_with_source` right here (to be sure that `cl_intel_reqd_subgroup_size` is supported) and check that value returned via `compile_sub_group_size` is correct

Reply:
I think it would be better to create another PR for this request.

Reply:
@AlexeySachkov I agree that current is enough to cover existence of compile_sub_group_size parameter.
We still should create separate test to check thatreqd_sub_group_size attribute works well .

Reply:
I'm working on it. Hopefully I'll submit the PR by EOD.

Reply:
Ok, works for me. Thanks!